### PR TITLE
Add scope and agenda

### PR DESCRIPTION
**Pull request by c-martinez**
Scope & draft agenda added.

### CODE REVIEW

File: content/sections/wosss21-agenda.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# WoSSS21 programme
+
```
Comment (shoaibsufi) [inline content/sections/wosss21-agenda.md]:
```suggestion
All times stated are in BST which is UTC+1
```

**Review by shoaibsufi — CHANGES_REQUESTED** [review]
Hi Carlos,

I suggested on addition for the agenda about timezones.

**Comment by c-martinez** [comment]
Thanks @shoaibsufi -- I've integrated your suggestion. I guess we can merge?

**Comment by shoaibsufi** [comment]
> Thanks @shoaibsufi -- I've integrated your suggestion. I guess we can merge?

I think you accept the change and then put it out for review again and then we merge but it's a lot of faff - so happy for you to merge once you have added the change.

**Pull request merged**